packages/maskbook/src/extension/service.ts

```diff
@@ -32,6 +32,7 @@ export const Services = {
     Ethereum: add(() => import('./background-script/EthereumService'), 'Ethereum'),
     SocialNetwork: add(() => import('./background-script/SocialNetworkService'), 'SocialNetwork'),
     Settings: add(() => import('./background-script/SettingsService'), 'Settings'),
+    Cloud: add(() => import('./background-script/CloudService'), 'Cloud'),
```

Comment:
Not necessary to add a new service entry for this feature. Use PluginServices instead.